docs/peer-validation.md

```diff
@@ -0,0 +1,32 @@
+# Peer Validation
+
+A [ValidatingAdmissionWebhook](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#validatingadmissionwebhook) can  be used to avoid applying faulty Peer configurations to the cluster.
+
+## How It works
+
+A ValidatingAdmissionWebhookConfiguration specifies a resource group (Peers), a service (the actual webhook) and operations (UPDATE, CREATE).
+After applying the configuration, the Kuberntes API server is going to send an AdmissionReviewRequest to the webhook service every time a Peer gets updated or created.
```

Comment:
If you want to be generic, than update or create are not the only possible operations. I wanted to be more specific and only describe this specific configuration. This is why I wrote peer instead of resource.

Reply:
OK I'm happy with either way: either keep it specific to Kilo or make it generic. However, just to make things clearer to the user, if we chose to keep it specific to Kilo, I think we should add something at the beginning of this sentence like "In the of the Kilo, ...". I say this because the sentence immediately before this one introduces the concept of ValidatingWebhookConfigurations at a high level independently from Kilo so the sentence after makes it sounds like (to me) all VWCs affect Peer resources.

Reply:
sgtm

Reply:
lets keep the first paragraph generic